src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -257,14 +257,7 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 					ShowInSearchPage = true,
 					ShowItem = itemsSelected && showOpenItemWith
 				},
-				new ContextMenuFlyoutItemViewModel()
-				{
-					Text = "OpenFileLocation".GetLocalizedResource(),
-					Glyph = "\uE8DA",
-					Command = commandsViewModel.OpenFileLocationCommand,
-					ShowItem = itemsSelected && selectedItems.All(i => i.IsShortcut),
-					ShowInSearchPage = true,
```

Comment:
Will this still show in the search page?

Reply:
I don't know, shortcuts do not appear in search results

Reply:
Search results can be from multiple locations so this allows users to open the location of the item.

Reply:
In stable the command that appears is open parent folder, not open file location

Reply:
Is that a separate action?

Reply:
Yes